parlai/core/params.py

```diff
@@ -738,6 +738,19 @@ def add_parlai_args(self, args=None):
             choices={None, 'full', 'batchsort'},
             help='Use dynamic batching',
         )
+        parlai.add_argument(
+            '-v',
+            '--verbose',
+            dest='verbose',
+            action='store_true',
```

Comment:
use `type="bool"` instead of `action="store_true"`. The problem with store true is that once we load it from an opt file, it's hard to unset.